### ISSUE TITLE
Polygons v/s MultiPolygons

### ISSUE THREAD

**Issue by bhaskarvk** [PitchInteractiveInc/hexagon-cartograms#79]
The exported topojsons use 'polygons' instead of 'multipolygons' for grouping related polygons.
E.g. congressional districts in a state, are represented as polygons of the state, but to be accurate they should be multipolygons, because as per topojson specs 

```
POLYGON((547.469447069539 794.993197302857, 564.05943028994 785.414965976816, 580.649413624811 794.993197302857, 580.649413624811 814.149659841916, 564.05943028994 823.727891167956, 547.469447069539 814.149659841916, 547.469447069539 794.993197302857), (530.879463849138 766.258503437758, 547.469447069539 756.680272111717, 564.05943028994 766.258503437758, 564.05943028994 785.414965976816, 547.469447069539 794.993197302857, 530.879463849138 785.414965976816, 530.879463849138 766.258503437758))
```

Is a a polygon where the first list is the outer ring and the subsequent lists are inner rings.

**Comment by jessihamel** [comment]
Yes, you're right. The exported topojsons should be encoded as multipolygons rather than polygons. I think this will also solve some issues I was seeing earlier with converting between topojson and geojson. We'll take a look into it soon and see if we can get a fix in.

**Comment by adamflorin** [comment]
Interesting. I can't say that I recall why it works the way it does. Thanks, @jessihamel for looking into it!

**Comment by jessihamel** [comment]
This issue should be fixed by #80. Let me know if you have any issues with the new encoding!
